pkgs/by-name/ti/tinymembench/package.nix

```
@@ -0,0 +1,28 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+}:
+
+stdenv.mkDerivation rec {
+  pname = "tinymembench";
+  version = "0.4";
+
+  src = fetchFromGitHub {
+    owner = "ssvb";
+    repo = "tinymembench";
+    rev = "v0.4";
+    hash = "sha256-N6jHRLqVSNe+Mk3WNfIEBGtVC7Y6/sERVaeAD68LQJc=";
+  };
+
+  installPhase = ''
+    install -D tinymembench $out/bin/tinymembench
```

```suggestion
    runHook preInstall
    install -D tinymembench $out/bin/tinymembench
    runHook postInstall
```
https://nixos.org/manual/nixpkgs/stable/#sec-stdenv-phases